pipelines/_ingredients/LSSTCam-imSim/DRP.yaml

```diff
@@ -69,7 +69,7 @@ tasks:
       python: |
         from lsst.pipe.tasks.diff_matched_tract_catalog import MatchedCatalogFluxesConfig
         columns_flux = {}
-        for band in 'ugrizy':
+        for band in ("u", "g", "r", "i", "z", "y"):
```

Comment:
If this is preferable, do you want to do it at lines 23 & 24 above?
```
        config.astromRefObjLoader.filterMap = {band: 'lsst_%s_smeared' % (band) for band in 'ugrizy'};
        config.photoRefObjLoader.filterMap = {band: 'lsst_%s_smeared' % (band) for band in 'ugrizy'};
```

Reply:
Better yet, I think I'll put the filterMap in parameters now that those can be referenced in python blocks.